src/lib/fields.ts

```diff
@@ -194,26 +178,26 @@ export async function referenceToRecord(str) {
     return null;
   }
 
-  return await RecordClass.select("id", "referenceNum").find(
+  // @ts-ignore
+  return await RecordClass.select(["id", "referenceNum"]).find(
     ahaReference.referenceNum
   );
```

Comment:
Can we use `findBy()` here to avoid throwing the UI alert if the record doesn't exist? (Returns the record or `undefined` if not found) 

```suggestion
  return await RecordClass.select(["id", "referenceNum"]).findBy({
    id: ahaReference.referenceNum
  });
```

Reply:
I don't want to make this functional change right now